clipboard_viewer/lib/main.dart

```diff
@@ -1,8 +1,8 @@
 import 'package:flutter/material.dart';
```

Comment:
oh remove this whole folder actually, this was a stupid test i made, not needed here

Reply:
the whole clipboard viewer directory?

Reply:
Ya

Reply:
done